app/services/qa/linked_data/authority_url_service.rb

```diff
@@ -35,13 +37,30 @@ def action_request_variable(action_config, action)
             action_config.qa_replacement_patterns[key]
           end
 
+          def supports_subauthorities?(action_config)
+            action_config.supports_subauthorities?
+          end
+
           def action_subauth_variable(action_config)
             action_config.qa_replacement_patterns[:subauth]
           end
 
           def action_subauth_variable_value(action_config, subauthority)
             action_config.subauthorities[subauthority.to_sym]
           end
+
+          def supports_language_parameter?(action_config)
+            action_config.supports_language_parameter?
+          end
+
+          def action_language_variable(action_config)
+            action_config.qa_replacement_patterns[:lang]
+          end
+
+          def language_value(language)
+            return nil if language.blank? || !language.is_a?(Array)
```

Comment:
@elrayle Just checking the logic here.  If the language is 'EN' you will return nil, but if the language is ['EN'] you will return 'EN' . Is this correct?

Reply:
Will return nil when language is...  nil, :en, 'en'
Will return :en when language is... ['en']

The reason this is ok is because the parameter is expected to be an array of symbols (line 12)
```
         # @param language [Array<Symbol>] languages for filtering returned literals (optional) 
```

The language is normalized to be an `Array<Symbol>` by the [language_service](https://github.com/samvera/questioning_authority/blob/5cd213de71a10db79f5914ee6524392ee07a2cbd/app/services/qa/linked_data/language_service.rb#L24-L29).

I could raise an exception if language is not `nil` or an `Array<Symbol>` to make this clearer.

Reply:
Went with removing `|| !language.is_a?(Array)` since the param doc says it should be an array.